main/src/library/Console.flix

```diff
@@ -91,6 +91,25 @@ namespace Console {
                 esc + "[38;2;1m" + ToString.toString(s) + esc + "[0m"
         }
 
+    ///
+    /// Returns `s` formatted with color `hexCode`.
+    ///
+    pub def hexColor(hexCode: String, s: a): String & Impure with ToString[a] =
+        import static java.lang.Integer.parseInt(String, Int32);
+
+        let hexNumbers = String.replace(from = "#", to = "", hexCode);
```

Comment:
Should we be enforcing a stricter format? This accepts e.g. `1#2#AB############34`

Reply:
Maybe it would be better to use String.stripPrefix. Then this would return `Option`, which could solve the problem of returning the original string.

Reply:
Yes, that would work.